.github/workflows/infrastructure-cleardown.yaml

```diff
@@ -0,0 +1,103 @@
+name: Cleardown Infrastructure
+
+on:
+  workflow_call:
+    inputs:
+      github_environment:
+        description: "Defines the Github environment in which to pull environment variables from"
+        required: true
+        type: string
+      branch_name:
+        description: "Name of the branch used to derive tf version and workspace"
+        required: true
+        default: 'not-set'
+        type: string
+
+jobs:
+  echo-workflow-inputs:
```

Comment:
Not sure but does this give us anything that we wouldn't get by calling meta data as you do below? If metadata doesn't output something you'd  find useful consider adding it there rather than this extra job?